nucypher/policy/conditions/lingo.py

```diff
@@ -60,13 +61,16 @@ def __init__(self, comparator: str, value: Union[int, str]):
         self.value = value
 
     def sanitize(self, comparator: str, value: str) -> Tuple[str, str]:
+        # Note that we're not sanitizing the value
         if comparator not in self.COMPARATORS:
             raise ValueError(f'{comparator} is not a permitted comparator.')
         return comparator, value
```

Comment:
Any reason we don't do `ast.literal_eval(value)` here, during `sanitize` at creation time, instead of doing it during `eval(...)`?

Reply:
Good point. It makes sense. I guess I was going it later for symmetry, since we need to validate also the `data` that's evaluated on the condition, but the sooner the better